types/libxmljs/index.d.ts

```diff
@@ -41,7 +41,7 @@ export class Document {
     encoding(): string;
     encoding(enc: string): this;
     find(xpath: string): Element[];
-    get(xpath: string): Element|null;
+    get(xpath: string, namespaces?: { [key: string]: string; }): Element|null;
```

Comment:
I see the type `{ [key: string]: string; }` in multiple places in this file. Is is possible to outsource it, e.g. define `NamespaceLiteral` (or any other name) once and then only use that name everywhere? Namely, without exposing `NamespaceLiteral` to consumers of this typing package.

I think I tried this once months ago and could not find a way to do it.